src/bin/pgcopydb/dump_restore.c

```diff
@@ -567,6 +570,167 @@ copydb_target_finalize_schema(CopyDataSpec *specs)
 }
 
 
+typedef struct ArchiveContext
+{
+	CopyDataSpec *specs;
+	PQExpBuffer line;
+	FILE *listFile;
+} ArchiveContext;
```

Comment:
This is the same name as in `archive_iterator.c` but with a different definition. While the C compiler has no problem dealing with that, I do. Please rename.